### DISCUSSION TITLE
Launching through steam does not work.

### DISCUSSION THREAD

**Discussion by Hoovenstein** [piotrulos/MSCModLoader#212] [Help me] [answered]
When hearing about the experimental Linux Support, I got my hopes up, as I had switched to Debian from Windows 10 back in June of this year. Earlier today, I read the guide on how to set up the Linux support. Everything wen't smoothly, I gave the files and folders correct permissions with chmod, etc. But when I went to launch the game through steam, it did not work. All that would happen is that when I clicked "PLAY", it turned blue and gave the option to "CANCEL", and my steam profile activity would say I'm playing the game for a split second before going back to it's previous state and not launching. 

I could start the game manually by launching the mysummercar.x86_64, and it would launch just fine(Didn't find my save files though), but it would not launch through steam Any ideas on what could be going wrong here? Because I did everything the guide said, so I don't think I'm missing anything. 

Any help would be appreciated, thanks.

**Comment by piotrulos** [comment, marked as answer]
Make sure you have in compatibility tab you selected mscloader native, and the compatibility .sh scrpts points to correct folder.

**Reply by Hoovenstein** [reply]
Seems to launch fine now through steam, shows I'm online, I can use overlay, take screenshots etc. Thanks. I'll leave a comment if I have any other issues, but it seems to work A-okay.

**Reply by Hoovenstein** [reply]
Yep. All works okay. Got modloader working. Thanks Again!
